gettsim/data/param.yaml

```
@@ -868,9 +871,36 @@ a2kiz_minek_sin:
     2005-01-01:
       value: 0
       note:
-    2009-01-01:
+    2008-10-01:
       value: 600
-      note:
+      note: G. v. 24.09.2008 BGBl. I S. 1854
+a2kiz_withdrawal_rate:
+  group: kiz
+  name:
+    de: Rate, mit der relevantes Einkommen auf den Kinderzuschlag angerechnet wird. Bis 07/2019 wurde das Einkommen nur in 10€-Schritten berücksichtigt.
```

```suggestion
    de: Minderungsrate durch Einkommen der Eltern über BMG
```
moved the long description to the description de key and added a suggestion for a short name here.